fossunited/hooks.py

```diff
@@ -71,6 +71,10 @@
         "from_route": "/events/<event>/rsvp/new",
         "to_route": "/rsvp/create/new",
     },
+    {
+        "from_route": "/events/<event_permalink>/schedule",
```

Comment:
I think along with just this redirect - adding a UI component (a button) on the event page is helpful. 

As the routes cannot be treated as obvious.

Reply:
Not a priority, since we are treating this as a dead page. Will add it with another PR which refactors `Schedule` tab in event page.